main.py

```diff
@@ -0,0 +1,90 @@
+"""
+This script monitors specified stocks, compares their prices with historical averages,
+and sends email notifications when certain conditions are met.
+"""
+
+#Import the necessary libraries and modules
+import yfinance as yf
+import smtplib
+import schedule
+import time
+import os
+from forex_python.converter import CurrencyRates
+from dotenv import load_dotenv
+from statistics import mean
+
+# Load environment variable from .env file
+load_dotenv()
+
+# Initialize currency converter
+currency_converter = CurrencyRates()
```

Comment:
Thats cool, never thought to get a currency converter library. Saves lines of code 👍